generative_ai/tuning.py

```diff
@@ -44,26 +46,27 @@ def tuning(
     Args:
       project_id: GCP Project ID, used to initialize vertexai
       location: GCP Region, used to initialize vertexai
-      training_data: GCS URI of training file or pandas dataframe of training data
+      training_data: GCS URI of jsonl file or pandas dataframe of training data
       train_steps: Number of training steps to use when tuning the model.
     """
-    vertexai.init(project=project_id, location=location)
+    vertexai.init(
+        project=project_id,
+        location=location,
+        credentials=credentials
+    )
     model = TextGenerationModel.from_pretrained("text-bison@001")
 
     model.tune_model(
-      training_data=training_data,
-      # Optional:
-      train_steps=train_steps,
-      tuning_job_location="europe-west4",
-      tuned_model_location="us-central1",
+        training_data=training_data,
+        # Optional:
+        train_steps=train_steps,
+        tuning_job_location="europe-west4",
```

Comment:
Should `tuning_job_location` be pulled out as a constant, or a default parameter?